playbooks/test_run_cukinia-sec.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2022, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+# Ansible playbook that runs Cukinia tests and export the result in CSV.
+
+---
+- hosts: cluster_machines
+  name: Cukinia tests
+  vars:
+    tests_format: junitxml
+    tests_result_name: cukinia.xml
+    cukinia_test_prefix: ".."
+    cukinia_configuration: "/etc/cukinia/cukinia-sec.conf"
+  tasks:
+    - name: Create temporary Cukinia directory
+      tempfile:
+        state: directory
+      register: tmp_cukinia_dir
+    - name: Run tests
+      shell:
+        cmd: >-
+          MACHINENAME={{ cukinia_namespace | default(inventory_hostname) }}
+          cukinia -f {{ tests_format }}
+          -o {{ tmp_cukinia_dir.path }}/{{ tests_result_name }}
+          {{ cukinia_configuration }} || true
+    - name: Fetch result
+      fetch:
+        src: "{{ tmp_cukinia_dir.path }}/{{ tests_result_name }}"
+        dest: "{{ cukinia_test_prefix }}/{{ tests_result_name }}"
+        flat: yes
+  run_once: true
```

Comment:
Why run this only in one machine?

Reply:
It was already here in previous version of test_run_cukinia. I removed it